app/src/components/steps/add-population-data-step.tsx

```diff
@@ -243,7 +252,9 @@ export default function SetPopulationDataStep({
                   mt={5}
                   mr={6}
                 >
-                  {!!year && <CheckIcon color="semantic.success" boxSize={4} />}
+                  {!!countryPopulationYear && countryPopulation && (
+                    <CheckIcon color="semantic.success" boxSize={4} />
+                  )}
```

Comment:
I wonder if this behavior could be extracted into a reusable component instead of copied a bunch of times for all inputs...

Same with the watch calls.

Reply:
That possible aswell

Reply:
I meant the whole input but this is a good start :+1: